desktop.css

```diff
@@ -1,248 +1,706 @@
+  color: #172b4d;
+  padding: 30px 10px;
+  overflow: hidden;
+  font-size: 16px;
+}
+
+.window-languages {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: flex-start;
+  max-width: 100%;
+  margin-bottom: 0;
+}
+
+.window-languages li {
+  color: #6070ff;
+  padding: 2px 12px;
+  margin: 0;
+  margin-right: 6px;
+  margin-bottom: 5px;
+  background-color: #ebebff;
+  border-radius: 8px;
+  font-size: 12px;
+  margin-top: 5px;
+}
+
+.window-btn-links {
+  display: flex;
+  justify-content: space-between;
+}
+
+.window-btn-links img {
+  color: white;
+}
+
+.window-btn-links img .up {
+  margin-left: 10px;
+}
+
+/* my portfolio works section */
+
+.card-container {
+  margin: 50px auto;
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-template-rows: 1fr 1fr 1fr 1fr;
+  gap: 100px 0;
+  grid-template-areas:
+    'card-1 card-1 card-1 card-1'
+    'card-2 card-2 card-2 card-2'
+    'card-3 card-3 card-3 card-3'
+    'card-4 card-4 card-4 card-4';
+  justify-items: center;
+  align-items: center;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  padding: 16px;
+  width: 330px;
+  margin: 5rem auto;
+  min-height: 570px;
+  background: #fff;
+  border: 1px solid #dfe1e6;
+  box-sizing: border-box;
+  border-radius: 15px;
+}
+
+.card-1 {
+  grid-area: card-1;
+}
+
+.card-2 {
+  grid-area: card-2;
+}
+
+.card-3 {
+  grid-area: card-3;
+}
+
+.card-4 {
+  grid-area: card-4;
+}
+
+.card img {
+  transition: all 0.4s ease-out;
+  height: 243px;
+  width: 296px;
+  object-fit: contain;
+}
+
+.card img:hover {
+  transform: scale(1.2);
+}
+
+.card h2 {
+  font-size: 32px;
+  font-style: normal;
+  font-weight: 700;
+  line-height: 44px;
+  letter-spacing: 0;
+  text-align: left;
+  color: #172b4d;
+  width: 60%;
+  margin-top: 10px;
+}
+
+.canopy {
+  display: flex;
+  width: 100%;
+  color: #172b4d;
+  margin: 14px 0;
+}
+
+.canopy h4 {
+  text-transform: uppercase;
+  font-size: 13px;
+  width: 25%;
+}
+
+.canopy ul {
+  list-style-type: disc;
+  display: flex;
+  flex: 1;
+  height: 10px;
+}
+
+.languages li {
+  color: #6070ff;
+  padding: 2px 12px;
+  margin-right: 6px;
+  border-radius: 8px;
+  background: #ebebff;
+  font-size: 12px;
+}
+
+.languages .about-list {
+  margin-top: -35px;
+  background: none;
+}
+
+.canopy ul li {
+  font-size: 13px;
+  margin-left: 3px;
+  line-height: 2px;
+  font-weight: 700;
+  color: #7a869a;
+}
+
+.card-descript {
+  height: 76px;
+  text-align: left;
+  color: #323c4e;
+  font-size: 15px;
+  display: -webkit-box;
+  max-width: 295px;
+  -webkit-line-clamp: 3;
+  -webkit-box-orient: vertical;
+  overflow: hidden;
+}
+
+.languages {
+  margin-top: 10px;
+  width: 300px;
+  display: flex;
+  justify-content: start;
+}
+
+button,
+.link {
+  width: 125px;
+  height: 50px;
+  color: #6070ff;
+  border: 2px solid #6070ff;
+  background-color: #fff;
+  border-radius: 8px;
+  margin-top: 25px;
+  transition: all 0.4s ease-out;
+}
+
+.window-btn-links .link {
+  width: 48%;
+}
+
```

Comment:
- [ ] Also make your "View Live" and "View Source"  has the same padding as you can see in the Figma image below 👇🏻  🤔 I think you can reduce or delete the `width: 48%;'  and use padding instead?

- Your Design:

![button32](https://user-images.githubusercontent.com/79058364/170544698-d41c8c67-3c53-401f-a300-dec2d00acdfe.png)

Reply:
Addressed ✔😉 

- [OPTIONAL] Great job so far 👏🏻 what do you think about adding padding or space between the `See live, See source` text and `icons` to make sure your design looks like the original one 💯 👍🏻 

- Your Design:

![butt52](https://user-images.githubusercontent.com/79058364/170672267-f55de133-d1c0-475c-ba2e-624692e6a5d8.png)